Makefile

```diff
@@ -3,19 +3,20 @@
 SUBDIRS = source/man
 
 # You can set these variables from the command line.
-SPHINXOPTS    += -W
+O            ?= -t man
+SPHINXOPTS   += -W
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+.PHONY: manpages help Makefile
```

Comment:
not sure what `manpages` helps here now?

Reply:
it's a leftover, ty !